docker-compose.yml

```diff
@@ -1,17 +1,38 @@
-version: '3.1'
+version: '3.4'
 
 services:
+    frontend:
+        build:
+            context: .
+            dockerfile: ./Dockerfile.frontend.dev
+        ports:
+            - 5001:5001
+    chatserver:
+        build:
+            context: .
+            dockerfile: ./Dockerfile.dev
+        ports:
+            - 8080:8080
+        depends_on:
+            - postgrest
     postgres:
         image: postgres:latest
         ports:
-            - 127.0.0.1:5432:5432
+            - 5432:5433
```

Comment:
@jimmcgaw I think these should be reversed, yes? To avoid port collisions on the server.

Reply:
Yep, got these backwards. 🙅